frontend/public/components/service-instance.tsx

```diff
@@ -73,7 +103,20 @@ const ServiceInstanceDetails: React.SFC<ServiceInstanceDetailsProps> = ({obj: si
   </React.Fragment>;
 };
 
-const pages = [navFactory.details(ServiceInstanceDetails), navFactory.editYaml(), navFactory.events(ResourceEventStream)];
+const ServiceBindingsDetails: React.SFC<ServiceBindingsDetailsProps> = ({obj: si}) => {
+  const bindingCreateHandler = createBinding(null, si).callback;
```

Comment:
I think it's a little cleaner to create a separate helper that is used by both the cog action and the callback here.

```tsx
const goToCreateBindingPage = (serviceInstance: K8sKindResource) => {  
  history.push(`/k8s/ns/${serviceInstance.metadata.namespace}/serviceinstances/${serviceInstance.metadata.name}/create-binding`);
};

const createBinding = (kindObj, serviceInstance) => ({
  callback: () => goToCreateBindingPage(serviceInstance),
  label: 'Create Service Binding',
});

// ...

<ServiceBindingsPage createHandler={() => goToCreateBindingPage(si)} ... />
```